### ISSUE TITLE
CD SEEK NORMAL error on certain games.

### ISSUE THREAD

**Issue by PepAlacant** [MiSTer-devel/TurboGrafx16_MiSTer#138]
Games like L-DIS or Sidearms Hyperdine will stop loading if CD SEEK is in NORMAL.

**Comment by dshadoff** [comment]
Please provide more information
- what version of the core ?
- what version of the main Mister ?
- what other settings are you using ?
- When you say "will stop loading", could you describe what the exact sequence of steps is, to reproduce the error ?

**Comment by PepAlacant** [comment]
Last version, last main.

it loads, but when you are about to start the game. If freezes.

**Comment by dshadoff** [comment]
Works fine for me on both games....
But based on the information above, I can't confirm whether my "last version" is the same as yours, or that I'm starting the games in the same way as you.

**Comment by PepAlacant** [comment]
Whe pressing start select options then try to start the game. If it works for you, i will try other iso set.

**Comment by dshadoff** [comment]
OK, I reverted to the current release of Main-MiSTer, and was able to recreate the issue on L-Dis by:
- selecting "Normal" difficulty
- then, on the weapons system screen, choosing the middle one.
In this case, I viewed the log output and saw the same output as I saw in Tenshi no Uta II's issue.

I then went back to the (not yet released) version from this morning, and couldn't recreate the issue.

So, it is solved - but not yet released.
